package.json

```diff
@@ -17,19 +17,18 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "prepublishOnly": "ember ts:precompile",
     "postpublish": "ember ts:clean",
-    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
     "test": "npm-run-all lint test:*",
     "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each",
```

Comment:
What runs `ember try:each` now?

Reply:
@scalvert it's done via `try-scenarios` - each scenario gets own task/job

My understanding that there is no need to do `ember try:each` as it runs tests in sequence (which was usually done in Travis) as we can dynamically create jobs in GH actions config

Reply:
@scalvert note this PR does not change the setup, it only fixes the mistake I made in #1164 where replaced `run: yarn test:ember` with `run: yarn test` (which did include `test:ember-compatibility` script which is now not part of addon blueprint)

Reply:
Got it. I was being a bit lazy about looking, so that's for letting me know.